packages/tools/src/stateManagement/annotation/annotationState.ts

```diff
@@ -40,58 +39,59 @@ function getViewportSpecificAnnotationManager(
   return defaultFrameOfReferenceSpecificAnnotationManager;
 }
 
+function getAnnotationManager(element?: HTMLDivElement) {
```

Comment:
Is there a good reason not to just use the annotation manager - if they need different managers, then probably the annotation itself should specify which manager, and not the viewport - otherwise, it just isn't clear how one would select the right manager.

Reply:
`getAnnotations` and `getAnnotation` both can provide the `element` optionally, which then here if we have a element (viewport)-specific one, it can grab from there. Identifying the manager inside the annotation doesn't seems right to me. It should be separate. The same annotation can live in five different managers IMO